packages/cli/src/cog/builder.ts

```diff
@@ -119,7 +121,7 @@ export class CogBuilder {
         const [resX] = image.resolution;
         let z = 30;
         while (z > 0) {
-            const currentZoom = TileSize / (1 << z);
+            const currentZoom = InitialResolution / (1 << z);
```

Comment:
might be a idea to leave this as a function and just call said function

Reply:
This logic is all going to change once we incorporate NZTM2000

Reply:
its more to prevent bugs like this :)